packages/frontend/messages/en.json

```diff
@@ -192,7 +192,10 @@
         "congratulations": "Congratulations",
         "launched": {
             "standard": "The campaign was successfully created.",
-            "safe": "The campaign creation transactions bundle is now ready to be signed. Please notify the other Safe participants."
+            "safe": {
+                "1": "The campaign creation transactions bundle is now ready to be signed.",
+                "2": "Please notify the other Safe participants"
```

Comment:
Missing final dot here?

Reply:
Oh right, addressed in 5709913e2486bbb7160ec6d865e30652380491cb